src/pages/actions/ledgerActionStore.ts

```diff
@@ -0,0 +1,42 @@
+export interface LedgerActionStore {
+  storeAction(actionName: string, swapId: string, transactionId?: string): void;
```

Comment:
:confused: I find the name of this method not very expressive. What we are trying to do is store, whether the action was already executed. 

- `markActionAsExecuted`

:clown_face: I just had a fun idea:

```ts
export interface LedgerActionMemory {
   rememberActionExecution(actionName: string, swapId: string, transactionId?: string);
   wasActionAlreadyExecuted(actionName: string, swapId: string): boolean;
}
```

Whatever you roll with, I think the name should express, what we are trying to achieve: Remembering that we already executed an action.

Reply:
I like your fun idea! I did struggle for a bit with this name and wasn't thrilled about it, so thank you for giving it some thought.